DEPENDENCIES.md

```diff
@@ -3,6 +3,7 @@
 | Name                      | Version          | Platform |                              Optional |
 | ------------------------- | ---------------- | -------- | ------------------------------------- |
 | autoconf                  | >= 2.69          |    All   |                                    No |
+| bc                        | any              |    All   |                  Yes (coverage in CI) |
```

Comment:
Is this really used?

Reply:
It is used in the `coverage.sh` script (see https://github.com/hyrise/hyrise/blob/52985d5547be37eefacc2f922f27b5092cecb7be/scripts/coverage.sh#L76)

Reply:
Got it, so we missed it till now. Not sure if that's needed though but that's nothing for this PR.